.github/workflows/build.yml

```diff
@@ -30,12 +30,16 @@ jobs:
         plat: [windows, uwp] # TODO: Support gamecore_console
         os: ['windows-2022']
         arch: [x86, x64, arm64]
-        tls: [schannel, quictls]
+        tls: [schannel, quictls, openssl]
         static: ['', '-Static']
         exclude:
         # OpenSSL/quictls doesn't support arm64
         - tls: quictls
           arch: arm64
+        - tls: openssl
+          arch: arm64
```

Comment:
Can we get arm64 working?

Reply:
openssl builds on arm64, there should be no reason this won't work, I had just assumed it was a problem with msquic, but I can drop that